core/domain/skill_jobs_one_off_test.py

```diff
@@ -138,11 +138,8 @@ def test_migration_job_converts_old_skill(self):
         }
         self.save_new_skill_with_defined_schema_versions(
             self.SKILL_ID, self.albert_id, 'A description', 0,
-            misconceptions=[], skill_contents=skill_contents)
-        skill = (
```

Comment:
@aks681, FYI I removed this code as `get_skill_by_id` also performs the migration and updates the schema version.

Reply:
Oh, ok, thanks!